### PR TITLE
PEP cleanup Part V

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1224,7 +1224,7 @@ def update_urns(self, user, urns):
                 priority -= 1
 
         # detach any existing URNs that weren't included
-        urn_ids = [urn.pk for urn in (urns_created + urns_attached + urns_retained)]
+        urn_ids = [u.pk for u in (urns_created + urns_attached + urns_retained)]
```
Comment (rowanseymour) [inline temba/contacts/models.py]: `urn` shadows var name above

File: temba/contacts/tests.py [hunk 2]
```diff
@@ -829,7 +829,8 @@ def test_contact_search(self):
         contact.set_field(self.user, 'isureporter', 'yes')
         contact.set_field(self.user, 'hasbirth', 'no')
 
-        q = lambda query: Contact.search(self.org, query)[0].count()
+        def q(query):
```
Comment (rowanseymour) [inline temba/contacts/tests.py]: Lambdas are cool until you start assigning them to variables, and then PEP wants you to use a `def` instead to help with stacktraces

File: temba/settings_common.py [hunk 3]
```diff
@@ -49,32 +48,31 @@
...
 # Default language used for this installation
-#-----------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------------
 LANGUAGE_CODE = 'en-us'
 
-#-----------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------------
 # Available languages for translation
-#-----------------------------------------------------------------------------------
-gettext = lambda s: s
```
Comment (rowanseymour) [inline temba/settings_common.py]: This was a workaround required back when `ugettext_lazy` couldn't be imported without circular dependencies.. but we already import `ugettext_lazy` above

File: temba/settings_common.py [hunk 4]
```diff
@@ -89,37 +87,11 @@
 # calendars according to the current locale
 USE_L10N = True
 
-# Absolute filesystem path to the directory that will hold user-uploaded files.
-# Example: "/home/media/media.lawrence.com/media/"
-MEDIA_ROOT = ''
```
Comment (rowanseymour) [inline temba/settings_common.py]: These were being redefined below in the directories section

**Comment by nicpottier** [comment]
woot

**Pull request merged**